raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -199,9 +239,9 @@ private void testElectionAfterLeaderFailure(QuorumConfig config) throws IOExcept
             scheduler.schedule(new SequentialAppendAction(cluster), 0, 2, 3);
             scheduler.runUntil(() -> cluster.anyReachedHighWatermark(10));
 
-            // Kill the leader and write some more data. We can verify the new leader has been elected
+            // Shutdown the leader and write some more data. We can verify the new leader has been elected
             // by verifying that the high watermark can still advance.
-            cluster.kill(1);
+            doShutdown.accept(cluster, 1);
```

Comment:
Hmm.. the leader's nodeId should be `0` not `1` right? I think the comment on line 230 is a bit misleading, we actually have nodes 0,1,2 where 0 is the leader.

Reply:
Ah, good catch! I had unintentionally fixed this and reverted the fix.